MaterialLib/PhysicalConstant.h

```diff
@@ -20,6 +20,9 @@ namespace PhysicalConstant
 /// Zero degrees Celsius in Kelvin
 const double CelsiusZeroInKelvin = 273.15;
 
+/// Room temperature, which is used as a reference temperature.
+const double RoomTemperature = CelsiusZeroInKelvin + 20.0;
```

Comment:
Better to call it nist_standard_temperature, or normal_temperature, or NTP_temperature;
Not this PR, but the CelsiusZeroInKelvin should be celsius_zero_in_kelvin too.

Reply:
Minor comment: All of PhysicalConstant.h is in CamelCase. IMHO we should keep that convention.

Reply:
Removed.